src/lib/qboard.ts

```diff
@@ -111,6 +114,7 @@ export default class QBoard {
     this.windowResize();
 
     window.onresize = this.windowResize;
+    window.onbeforeunload = () => this.isModified || null;
```

Comment:
wait what exactly does this do

Reply:
Provides the alert iff `this.isModified`. We have the choice of making the function null or making it return null, here we choose the second/

Reply:
IE is stupid and actually alerts whatever this returns. So if it returns null then the dialog contains "null." If it returns true then the dialog contains "true." Do we care?